authorization/handler.go

```diff
@@ -0,0 +1,99 @@
+package authorization
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/SKF/go-rest-utility/problems"
+	"github.com/SKF/go-utility/v2/log"
+	"github.com/SKF/go-utility/v2/useridcontext"
+	proto "github.com/SKF/proto/v2/common"
+	"github.com/gorilla/mux"
+
+	middleware "github.com/SKF/go-enlight-middleware"
+)
+
+type AuthorizerClient interface {
+	IsAuthorizedWithContext(ctx context.Context, userID, action string, resource *proto.Origin) (bool, error)
```

Comment:
Since we have no corresponding method without context, I think this should just be `IsAuthorized()`. The same goes for ` GetResourceWithContext()`.

Reply:
This interface is a subset of the real AuthorizerClient from the Enlight SDK.